Python/ceval.c

```diff
@@ -814,9 +816,9 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
         _Py_Instrument(_PyFrame_GetCode(frame), tstate->interp);
         monitor_throw(tstate, frame, frame->instr_ptr);
         /* TO DO -- Monitor throw entry. */
+        DTRACE_FUNCTION_ENTRY();
         goto resume_with_error;
     }
-
```

Comment:
Let's revert this change as well. That way we have a smaller diff (and also I think the new line here is good for readability).